src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/custom.py

```diff
@@ -48,15 +46,16 @@ def acr_create(registry_name, #pylint: disable=too-many-arguments
                resource_group_name,
                location,
                storage_account_name=None,
-               enable_admin=False):
-    '''Create a container registry.
+               admin_enabled='false'):
+    '''Creates or updates a container registry with the specified parameters.
```

Comment:
Same here. PUT doesn't really "update" so much as "overwrite existing", which is different from your actual update command.